android/src/main/java/com/zmxv/RNSound/RNSoundModule.java

```diff
@@ -184,6 +184,16 @@ protected MediaPlayer createMediaPlayer(final String fileName) {
         }
     }
 
+    if (fileName.startsWith("file:/")){
```

Comment:
I think this can be merged to [http(s) condition](https://github.com/zmxv/react-native-sound/blob/41ec9b91b02cc85fe1b14ae4234a65cedacbf004/android/src/main/java/com/zmxv/RNSound/RNSoundModule.java#L163) ?

Reply:
Hi @beaulieufrancois thanks for this change! Could you please address this comment? I’d like to merge this :)

Reply:
@paulmelnikow Since there is no response and yet this pr is quite important to many users, can you edit it directly or merge it first?

Reply:
Hi, I'm not working on this project anymore and am in no position to test this, though if you want to open a new PR that addresses this comment I'll merge it.

Reply:
@paulmelnikow Ok, I've made a new pr. https://github.com/zmxv/react-native-sound/pull/663